GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2053,6 +2053,14 @@ private string GetRefUnambiguousName(IGitRef gitRef)
                 : gitRef.Name;
         }
 
+        private void RebaseOnToolStripMenuItem_DropDownOpening(object sender, EventArgs e)
+        {
+            int selectedRevisionsCount = GetSelectedRevisions().Count;
+            rebaseToolStripMenuItem.Enabled =
+            rebaseInteractivelyToolStripMenuItem.Enabled = selectedRevisionsCount == 1;
+            rebaseWithAdvOptionsToolStripMenuItem.Enabled = selectedRevisionsCount == 1 || selectedRevisionsCount == 2;
```

Comment:
```suggestion
            rebaseWithAdvOptionsToolStripMenuItem.Enabled = selectedRevisionsCount is (1 or 2);
```
❓ `&& !IsArtificial`

Reply:
Yes, I had that is mind when I started working on it but I forgot. Fixed